regression/contracts/assigns_enforce_19/main.c

```diff
@@ -0,0 +1,33 @@
+#include <assert.h>
+
+static int b = 0;
+static int c = 0;
+
+int f() __CPROVER_assigns()
+{
+  static int a = 0;
+  a++;
+  return a;
+}
+
+int g(int *points_to_b, int *points_to_c)
+  __CPROVER_assigns(b) // Error: it should also mention c
+{
+  b = 1;
+  c = 2;
+  *points_to_b = 1;
```

Comment:
Nice test case! We are now correctly looking at assignable (and freely assignable) "addresses" instead of symbols :)